rotkehlchen/tests/unit/test_bitcoin.py

```diff
@@ -14,6 +17,7 @@
     UNIT_BTC_ADDRESS2,
     UNIT_BTC_ADDRESS3,
 )
+from rotkehlchen.tests.utils.substrate import ENS_BRUNO_BTC_ADDR, ENS_BRUNO_BTC_BYTES
```

Comment:
This should not be in utils.substrate. Has nothing to do with substrate. This should either be only on this file or on a bitcoin related test utils file.